### ISSUE TITLE
How to run the test set data?

### ISSUE THREAD

**Issue by dwdjsy89** [chow-vincent/tennis_action_recognition#2]
You wrote "we are able to achieve a performance of 82.3% on test set data", but how did you test your model?
We ran the training code but didn't have the code to test it.

**Comment by chow-vincent** [comment]
See the script `lrcnn_evaluate.py`
